common-test/src/main/java/io/strimzi/test/Topic.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.test;
+
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.lang.annotation.Repeatable;
+
+/**
+ * Annotation for test classes or methods run via {@code @RunWith(StrimziRunner.class)}
+ * which causes that runner to create kafka clusters before, and delete kafka clusters after,
+ * the tests.
+ */
+@Target({ElementType.METHOD, ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+@Repeatable(Topic.Container.class)
```

Comment:
I think that this annotation shouldn't be repeatable because I think we will not have cases when we will use it for class level.